inc/wing/Value.hpp

```diff
@@ -29,17 +30,17 @@ class Value {
      */
     auto IsNull() const -> bool;
     auto AsStringView() const -> std::string_view;
```

Comment:
shouldn't string_view also be wrapped in an optional?  The internal type might need to be optional itself and determined when the Value is created from either a nullptr or an actual char* + length?